test/install_test.go

```diff
@@ -655,7 +655,7 @@ func TestInject(t *testing.T) {
 }
 
 func TestServiceProfileDeploy(t *testing.T) {
-	bbProto, err := TestHelper.HTTPGetURL("https://raw.githubusercontent.com/BuoyantIO/bb/master/api.proto")
+	bbProto, err := TestHelper.HTTPGetURL("https://raw.githubusercontent.com/BuoyantIO/bb/55b78f210c98a7eb431c3f555cf0a406f0b12edf/api.proto")
```

Comment:
this might look nicer if there's a release tag we can target instead of an arbitrary looking sha

Reply:
Good idea. changed to the latest tagged version: `https://raw.githubusercontent.com/BuoyantIO/bb/v0.0.5/api.proto`